.github/ISSUE_TEMPLATE/new_chain.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Evaluate new chain
+about: Evaluation process for Safe contract deployment on new chains
+
+---
+
+## Useful Links
+
+Please list any useful links that you encounter in the process. Such as, block explorers, faucets, bridges, etc.
+
+## ChainId
+
+- Provide the chain ID: 
+- Provide the link to the [chainlist.org](https://chainlist.org) entry in their [repository](https://github.com/ethereum-lists/chains/tree/master/_data/chains) with the chain details: 
+
+## Validation steps
+
+As you complete the steps, provide links to the chain's block explorer transactions, to verify the correct execution of the steps.
+
+- [ ] If contracts are not deployed. Deploy them using [the tool](https://github.com/gnosis/safe-contracts)
+  - [ ] Verify the chain at https://chainlist.org/ .
+- [ ] Please document links of interest such as faucets, bridges, block explorer, etc. as comments in this issue.
+- [ ] Execute the steps in our internal [guide](https://app.gitbook.com/o/-MhyDtGxUyODu7d4sgda/s/-MhyEL1Pq_BGR0rFugaE/backend/contract-deployment-checks) for user side validation of the contracts:
```

Comment:
I would make this section optional, as it is more required for us before we test our services.

Reply:
Hi there..azim here..be glad if you can help me with that because im non technical coeing actually. Thanks sir